src/routes/(console)/+layout.ts

```diff
@@ -1,71 +1,69 @@
-import { Dependencies } from '$lib/constants';
-import type { Plan } from '$lib/sdk/billing';
-import type { Tier } from '$lib/stores/billing';
 import { sdk } from '$lib/stores/sdk';
 import { isCloud } from '$lib/system';
 import type { LayoutLoad } from './$types';
-import { Query, type Models } from '@appwrite.io/console';
+import { Query } from '@appwrite.io/console';
+import { Dependencies } from '$lib/constants';
+import type { Tier } from '$lib/stores/billing';
+import type { Plan, PlanList } from '$lib/sdk/billing';
+import { consoleVariables, version } from '$routes/(console)/store';
 
-export const load: LayoutLoad = async ({ params, fetch, depends, parent }) => {
+export const load: LayoutLoad = async ({ params, depends, parent }) => {
     await parent();
     depends(Dependencies.RUNTIMES);
     depends(Dependencies.CONSOLE_VARIABLES);
     depends(Dependencies.ORGANIZATION);
 
-    const prefs = await sdk.forConsole.account.getPrefs();
-
     const { endpoint, project } = sdk.forConsole.client.config;
-    const versionPromise = fetch(`${endpoint}/health/version`, {
-        headers: {
-            'X-Appwrite-Project': project
-        }
-    }).then((response) => response.json() as { version?: string });
-
-    const variablesPromise = sdk.forConsole.console.variables();
+    const [preferences, plansArray, organizations] = await loadPromises();
 
-    const [data, variables] = await Promise.all([versionPromise, variablesPromise]);
+    const plansInfo = toPlanMap(plansArray);
 
-    let plansInfo = new Map<Tier, Plan>();
-    if (isCloud) {
-        const plansArray = await sdk.forConsole.billing.getPlansInfo();
-        plansInfo = plansArray.plans.reduce((map, plan) => {
-            map.set(plan.$id as Tier, plan);
-            return map;
-        }, new Map<Tier, Plan>());
-    }
+    const currentOrgId =
+        params.organization ??
+        preferences.organization ??
+        (organizations.teams.length > 0 ? organizations.teams[0].$id : undefined);
 
-    const organizations = !isCloud
-        ? await sdk.forConsole.teams.list()
-        : await sdk.forConsole.billing.listOrganization();
+    fetch(`${endpoint}/health/version`, {
+        headers: { 'X-Appwrite-Project': project }
+    })
+        .then((res) => res.json().catch(() => null))
+        .then((data) => version.set(data?.version));
```

Comment:
same here we stores

Reply:
if we wanna put async data into stores, it should happen with the store, in the load functions 👍🏻